controls/def.cf

```diff
@@ -126,6 +126,14 @@ bundle common def
                           not(isvariable("trustkeysfrom"))),
         meta => { "defvar" };
 
+      "cf_serverd_maxconnections"
```

Comment:
Functions should work in bodies, yes. I haven't tested it, but if it doesn't work I'd consider that a bug.

I would make the variable `def.cfe_controls[server][maxconnections]` so we have just one new variable in `def`.